### PR TITLE
fix: re-enable online archive start command

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

 Looks like this command got lost at some point, adding it back along with some e2e testing 

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: e2e/E2E-TESTS.md [hunk 1]
```diff
@@ -160,10 +160,11 @@ Command                                         | E2E Atlas | E2E OM    | E2E CM
...
 | `config`                                      |           |           |           |           |       |       |
-| `config delete`                               |           |           |           |           |       |       |
-| `config list`                                 |           |           |           |           |       |       |
-| `config describe`                             |           |           |           |           |       |       |
-| `config rename`                               |           |           |           |           |       |       |
+| `completion`                                  | Y         | Y         | Y         | Y         | Y     | Y     |
+| `config delete`                               | Y         | Y         | Y         | Y         | Y     | Y     |
+| `config list`                                 | Y         | Y         | Y         | Y         | Y     | Y     |
+| `config describe`                             | Y         | Y         | Y         | Y         | Y     | Y     |
+| `config rename`                               | Y         | Y         | Y         | Y         | Y     | Y     |
```
Comment (gssbzn) [inline e2e/E2E-TESTS.md:167]: @andreaangiolillo not sure if this makes sense, they had e2e testing but they are not really atlas or ops manager test, they are generic ones

Comment (andreaangiolillo) [reply e2e/E2E-TESTS.md:167]: My idea was that this file would list all the commands and related e2e tests but I see that setting Y for `E2E Atlas | E2E OM    | E2E CM` may be confusing in this case. We could create a new column E2E-generic or remove them, I don't have a strong opinion about it

File: internal/cli/atlas/onlinearchive/online_archive_test.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+)
+
+func TestBuilder(t *testing.T) {
+	test.CmdValidator(
+		t,
+		Builder(),
+		7,
+		[]string{},
+	)
+}
```
Comment (andreaangiolillo) [inline internal/cli/atlas/onlinearchive/online_archive_test.go:32]: Could we add the `Test"Command"Builder `test also to the other commands?

Comment (gssbzn) [reply internal/cli/atlas/onlinearchive/online_archive_test.go]: if you insist, making this PR a bit larger tho

**Review by andreaangiolillo — DISMISSED** [review]
LGTM

**Comment by gssbzn** [comment]
@andreaangiolillo test are green finally

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**